common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java

```diff
@@ -211,9 +211,6 @@ public boolean hasImage() {
 	 * @return
 	 */
 	public int getPosition() {
-		if(position==-1 && getReplacementString()!=null) {

```

Comment:
That was a dead code. The method was not used at all. But now we need this method. So I cleaned it up before starting using it.